openslides_backend/services/media/interface.py

```diff
@@ -6,10 +8,13 @@ class MediaService(Protocol):
     Mediaservice defines the interface to the mediaservice.
     """
 
-    def upload(
+    def upload_mediafile(
         self,
         file: str,
         id: int,
         mimetype: str,
-    ) -> None:
+    ) -> Union[str, None]:
+        ...
+
+    def upload_resource(self, file: str, id: int, mimetype: str) -> Union[str, None]:
```

Comment:
Add a documentation about the error, that can be thrown.

Reply:
Done